docs/getting_started.md

```diff
@@ -0,0 +1,107 @@
+
+
+# Getting started with OVMS
+
+
+## OpenVino Model Server
+
+### Requirements
+* docker version >= 19.03
+* git version >= 1.8.3
+* wget version >= 1.14
+
+Docker Community Edition Information
+To install Docker Community Edition (CE), follow: official Docker instructions.
+The build process requires access to the docker command. Remember to add your user to the docker group by running: sudo usermod -aG docker [user] if the user has not been previously added.
+For more information, refer to the Post-install Docker guide.
+
+**1. Clone repository**  
+
+Assumption: Access to github.com based on ssh key (see: https://help.github.com/articles/connecting-to-github-with-ssh/)
+```
+git clone ssh://git@github.com:IntelAI/OpenVINO-model-server.git
+```
+Move to main repository directory - OpenVINO-model-server.
+
+**2. OVMS image preparation**  
+
+Download OVMS image using docker (from: https://hub.docker.com/r/intelaipg/openvino-model-server):
+```
+docker pull intelaipg/openvino-model-server
+```
+or build image via [make](https://github.com/IntelAI/OpenVINO-model-server#building):
+```
+make docker_build_apt_ubuntu
+```
+
+**3. Model preparation - resnet example**  
+
+Correct model directory structure:
+```bash
+models/
+├── model/
+    ├── 1/
+        ├── ir_model.bin
+        └── ir_model.xml
+```
+Prepare directory according to the structure above:
+```
+mkdir -p models/model/1/
+```
+Move to the created direcory:
+```
+cd models/model/1/
+```
+Download model files (.xml & .bin) - resnet example
+```
+wget https://download.01.org/opencv/2020/openvinotoolkit/2020.1/open_model_zoo/models_bin/1/resnet50-binary-0001/FP32-INT1/resnet50-binary-0001.bin
+```
+```
+wget https://download.01.org/opencv/2020/openvinotoolkit/2020.1/open_model_zoo/models_bin/1/resnet50-binary-0001/FP32-INT1/resnet50-binary-0001.xml
+```
+**4. Run OpenVino Model Server**  
+
+Start OVMS using docker - [details](https://github.com/IntelAI/OpenVINO-model-server/blob/master/docs/docker_container.md):
+```
+docker run --rm -d -v /path/to/models:/docker/models/path -p <port>:<port> <rest_port>:<rest_port> ie-serving-py \
+/ie-serving-py/start_server.sh ie_serving model --model_path /docker/models/path  --model_name <model_name> --port <port> --rest_port <rest_port> --grpc_workers 32  --nireq 32 --target_device CPU --batch_size 1 --plugin_config "{\"CPU_THROUGHPUT_STREAMS\": \"CPU_THROUGHPUT_AUTO\"}"
```

Comment:
Since it's "Getting Started" perhaps default settings are enough to make it as simple as possible? 
`grpc_workers`, `nireq` and `plugin_config` params are by default set for latency mode. 
`batch_size` and `target_device` are set to 1 and"CPU" by default.
So:
`/ie-serving-py/start_server.sh ie_serving model --model_path /docker/models/path  --model_name <model_name> --port <port> --rest_port <rest_port>`
shall be enough for simple CPU example